docs/source/AdministratorGuide/Tutorials/basicTutoSetup.sh

```diff
@@ -60,15 +66,15 @@ start runsvdir || restart runsvdir
 yum remove -y $(rpm -qa | grep -i mysql | paste -sd ' ')
```

Comment:
```suggestion
 yum remove -y $(rpm -qa | grep -i -e mysql -e mariadb | paste -sd ' ') || echo "nothing to remove?"
```
This ignores errors in this line, and also removed mariadb packages, which is what I had to do on my cc7 VM